python/cudf/cudf/core/dataframe.py

```diff
@@ -5494,14 +5503,9 @@ def from_pandas(cls, dataframe, nan_as_null=no_default):
             )
 
         if isinstance(dataframe, pd.DataFrame):
-            if not dataframe.columns.is_unique:
```

Comment:
Can you update the PR description to indicate that this PR also enables this on frames with duplicate column names?

Reply:
Hmm this removal doesn't actually change any behavior with duplicate column names (still raises but now below in the `df.column = dataframe.column` call)

Reply:
Got it, never mind then.